### DISCUSSION TITLE
difference between number of genes identified and pg matrix

### DISCUSSION THREAD

**Discussion by dachaotou** [vdemichev/DiaNN#186] [Q&A]
Hi, Vadim
Im chao，i have a question for you. In the log.txt, Number of genes identified at 1% FDR: 3698 (precursor-level), 3173 (protein-level) (inference performed using proteotypic peptides only), this number is quite different with the pg matrix file which is more than 3173 and reach to 4900 protein groups. 
i want to know why this two number is different and where can i find the list of Number of genes identified at 1% FDR,how this number count?

Thank you in advance!

Best,chao

**Comment by vdemichev** [comment]
Hi Chao,

Please see my last answer here https://github.com/vdemichev/DiaNN/issues/180.

Best,
Vadim

**Reply by vdemichev** [reply]
In short, there are two answers:
Super simple: if you are interested in gene groups, just use gg_matrix; if want unique genes, use the respective matrix
Also simple: use main report and filter either Global.PG.Q.Value <= 0.01 or GG.Q.Value <= 0.01

**Reply by dachaotou** [reply]
thanks a lot. problem solved.
